integration-tests/src/test/compose/docker-compose.yml

```
@@ -0,0 +1,41 @@
+version: "2.2"
+
+services:
+  alfresco:
+    image: ${DOCKER_IMAGE}
+    environment:
+      JAVA_OPTS: "
+                -Ddb.driver=org.postgresql.Driver
+                -Ddb.username=alfresco
+                -Ddb.password=alfresco
+                -Ddb.url=jdbc:postgresql://postgres:5432/alfresco
+                -Dindex.subsystem.name=noindex
+                -Djodconverter.enabled=false
+                -Ddeployment.method=DOCKER_COMPOSE
+                -Dcsrf.filter.enabled=false
+                -Xms1g -Xmx1g
+                -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8000
+                "
+    ports:
+    - ${COMPOSE_ALFRESCO_TCP_8080:-8080:8080} # by default expose on :8080
```

Shouldn't this be prefixed with `$DOCKER_IP`, so you can choose to bind to localhost or a specific IP?
```suggestion
    - $DOCKER_IP:${COMPOSE_ALFRESCO_TCP_8080:-8080:8080} # by default expose on :8080
```